### PR TITLE
Add course pages to learn

### PR DESCRIPTION
**Pull request by JonasAls89**

### CODE REVIEW

File: hub/sesam-training-course.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ 
+You will learn about:
+---------------------
+- What a data driven integration is and how it differs from previous integration architectures.
+- How to use the Sesam Management Studio.
+- How to collect, enrich, connect, transform and share data as it passes through Sesam towards its targets.
+ 
+Who should sign up:
+-------------------
+- Integration engineers and/or developers who would like to learn more about data driven architecture in addition to getting practice experience with it.
```
Comment (ga-hegsvold) [inline hub/sesam-training-course.rst:17]: Suggested rephrasing:
Integration engineers and developers who would like to learn more about, and get some hands-on experience with, data driven architecture.

File: hub/sesam-training-course.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- Architects who would like to learn how Sesam places itself technically in comparison with other integration solutions.
+ 
+Requirements:
+-------------
+- None, but prior experience with SQL and some coding will be beneficial.
+
+Course structure:
+-----------------
+The course is divided into smaller theoretical sections followed by practical examples to engage the course taker. This is used to ensure a tight coupling between theory and practice.
```
Comment (ga-hegsvold) [inline hub/sesam-training-course.rst:26]: "course taker" -> "course participant"

File: hub/sesam-training-course.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- Introduction
+- Integration architectures
+- Management Studio and Sesam concepts
+- Importing data and concepts continued
+- Datasets
+- Merging of data
+- Joins and intersections
+- Playing the long game
+
+*Day two: DTl in continuation and more complex integrations*
```
Comment (ga-hegsvold) [inline hub/sesam-training-course.rst:40]: "DTl" -> "DTL"

File: hub/working-in-sesam-projects.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
+=========================
+Working in Sesam Projects
+=========================
+
+Working in Sesam Projects is a two day course where you will learn to appreciate how Sesam should be used in Projects. As such, you will get to know how Sesam is hosted and what aspects of Sesam is important when setting up your project architecture with the use of tools in general.
```
Comment (ga-hegsvold) [inline hub/working-in-sesam-projects.rst:5]: "Projects" -> "projects"

File: hub/working-in-sesam-projects.rst [hunk 5]
```diff
@@ -0,0 +1,39 @@
+=========================
+Working in Sesam Projects
+=========================
+
+Working in Sesam Projects is a two day course where you will learn to appreciate how Sesam should be used in Projects. As such, you will get to know how Sesam is hosted and what aspects of Sesam is important when setting up your project architecture with the use of tools in general. 
+
+This course has a tight coupling between theory and practice and as a course taker you will get to experience and try each of the concepts that we go through.
```
Comment (ga-hegsvold) [inline hub/working-in-sesam-projects.rst:7]: "course taker" -> "course participant"

File: hub/working-in-sesam-projects.rst [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+You will learn about:
+---------------------
+- How Sesam can be hosted and the Sesam API. Then you will get to appreciate how Sesam uses version control and how the Sesam client is used in order to make development agile and support test driven development. Finally, you will learn about deployment processes and gitflow to ensure that you have all the pieces of information needed to properly set up a Sesam project.
```
Comment (ga-hegsvold) [inline hub/working-in-sesam-projects.rst:11]: I think this paragraph should be written as multiple bullet points, not just one big bullet point.
Maybe separate the topics something like this:
- Sesam hosting
- Sesam API
- Sesam Client and TDD
- Sesam with git to enable CI and deployment

File: hub/working-in-sesam-projects.rst [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+This course has a tight coupling between theory and practice and as a course taker you will get to experience and try each of the concepts that we go through.
+
+You will learn about:
+---------------------
+- How Sesam can be hosted and the Sesam API. Then you will get to appreciate how Sesam uses version control and how the Sesam client is used in order to make development agile and support test driven development. Finally, you will learn about deployment processes and gitflow to ensure that you have all the pieces of information needed to properly set up a Sesam project. 
+ 
+Who should sign up:
+-------------------
+- Integration engineers and/or developers who would like to learn how to work in Sesam projects in addition to getting practice experience with it.
```
Comment (ga-hegsvold) [inline hub/working-in-sesam-projects.rst:15]: Suggested rephrasing:
Integration engineers and developers who would like to learn more about, and get some hands-on experience with, working in Sesam projects.

**Review by ga-hegsvold — CHANGES_REQUESTED** [review]

**Review by ga-hegsvold — APPROVED** [review]

**Review by djalil-bouhachi — APPROVED** [review]

**Pull request merged**